File: UI/UI-Session11/ShoppingCart/src/app/entities/item.ts
export class item{
    public id:any;
    public name;
    public price;
    public description;
    public imgUrl;
    public category;

    constructor( id, name:string, price, description:string, imgUrl:string, category:string){
        this.id = id;
        this.name = name;
        this.price = price;
        this.description = description;
        this.imgUrl = imgUrl;
        this.category = category;
       
    }
}

File: UI/UI-Session11/ItemApi/src/main/java/com/metacube/model/Item.java
package com.metacube.model;

import javax.xml.bind.annotation.XmlRootElement;

/**
 * Item POJO
 * @author Vinod
 *
 */
@XmlRootElement
public class Item {
	private int id;
	private String name;
	private String price;
	private String description;
	private String imgUrl;
	private String category;
	
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getPrice() {
		return price;
	}
	public void setPrice(String price) {
		this.price = price;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getImgUrl() {
		return imgUrl;
	}
	public void setImgUrl(String imgUrl) {
		this.imgUrl = imgUrl;
	}
	public String getCategory() {
		return category;
	}
	public void setCategory(String category) {
		this.category = category;
	} 
	
	
}
